creator/studies/mutations.py

```diff
@@ -736,6 +736,75 @@ def mutate(self, info, study, data):
         return UpdatePhenotypeStatusMutation(study=study)
 
 
+class TransferStudyMutation(graphene.Mutation):
+    """ Mutation to transfer study to another organization"""
+
+    class Arguments:
+        study = graphene.ID(
+            required=True,
+            description="The ID of the study to remove the collaborator from",
+        )
+        organization = graphene.ID(
+            required=True,
+            description="The organization to transfer the study to."
+        )
+
+    study = graphene.Field(StudyNode)
+
+    def mutate(self, info, study, organization):
+        """
+        Update the organization of a study
+        """
+        user = info.context.user
+        if not user.has_perm("organizations.change_organization"):
+            raise GraphQLError("Not allowed")
+
+        try:
+            _, study_id = from_global_id(study)
+            study = Study.objects.get(kf_id=study_id)
+        except (Study.DoesNotExist):
+            raise GraphQLError(f"Study {study_id} does not exist.")
+
+        try:
+            _, organization_id = from_global_id(organization)
+            organizationNode = Organization.objects.get(pk=organization_id)
+        except Organization.DoesNotExist:
+            raise GraphQLError(
+                f"Organization {organization_id} does not exist"
+            )
+
+        if not user.organizations.filter(pk=study.organization.id).exists():
+            raise GraphQLError(
+                "Not allowed - may only transfer study from an "
+                "organization the user is a member of."
+            )
+        if not user.organizations.filter(pk=organization_id).exists():
+            raise GraphQLError(
+                "Not allowed - may only transfer study to an "
+                "organization the user is a member of."
+            )
```

Comment:
```suggestion
       org_msg = (
            "Not allowed - may only transfer study from an "
             "organization the user is a member of."
        )
        if not user.organizations.filter(pk=study.organization.id).exists():
            raise GraphQLError(org_msg)
        if not user.organizations.filter(pk=organization_id).exists():
            raise GraphQLError(org_msg)
```

Reply:
Here the message differences are `from an organization` and `to an organization`